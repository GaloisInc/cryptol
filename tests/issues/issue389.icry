:l issue389.cry
pad (join "a" : [8])
test (join "a" : [8])
