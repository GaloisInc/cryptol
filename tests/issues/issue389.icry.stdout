Loading module Cryptol
Loading module Cryptol
Loading module Main
0x61800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008
0x61800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008
